pkg/query/endpointset.go

```diff
@@ -228,7 +238,17 @@ func (c *endpointSetNodeCollector) Collect(ch chan<- prometheus.Metric) {
 			if storeType != nil {
 				storeTypeStr = storeType.String()
 			}
-			ch <- prometheus.MustNewConstMetric(c.connectionsDesc, prometheus.GaugeValue, float64(occurrences), externalLabels, storeTypeStr)
+			// select only required labels
```

Comment:
Use full sentence please.
```suggestion
			// Select only required labels.
```